src/content/release/breaking-changes/flutter-gradle-plugin-apply.md

```diff
@@ -155,8 +159,9 @@ tasks.register("clean", Delete) {
 
 ### android/app/build.gradle
 
-Most changes have to be made in the `<app-src>/android/app/build.gradle`. First,
-remove these 2 chunks of code that use the legacy imperative apply method:
+Most changes have to be made in `<app-src>/android/app/settings.gradle`.
```

Comment:
This section and its header are talking about `build.gradle`. Checking the framework, it does seem these snippets are usually found in the `build.gradle`, but maybe developers could have moved them for their own apps? So perhaps this could just adjust the text to say it's the usual destination? Something like:

```suggestion
The following changes are also needed, usually in `<app-src>/android/app/build.gradle`.
```